### PR TITLE
Update README to show correct creation of extension

### LINKED ISSUE #92: Change Readme Example
We should be using proper accessors on the container in our example.

```php
        $basePath = rtrim(str_ireplace('index.php', '', $container->get('request')->getUri()->getBasePath()), '/');
        $view->addExtension(new \Slim\Views\TwigExtension($container->get('router'), $basePath));

```

### PR DESCRIPTION
**Pull request by akrabat**
Create a `Uri` object rather than a string and also show usage of PSR-11
method to retrieve from container.

Closes #92

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,8 +30,9 @@ $container['view'] = function ($c) {
     ]);
     
     // Instantiate and add Slim specific extension
-    $basePath = rtrim(str_ireplace('index.php', '', $c['request']->getUri()->getBasePath()), '/');
-    $view->addExtension(new \Slim\Views\TwigExtension($c['router'], $basePath));
+    $router = $c->get('router');
+    $uri = Slim\Http\Uri::createFromEnvironment(new \Slim\Http\Environment($_SERVER)):
```
Comment (geggleto) [inline README.md]: Isn't the environment already registered in the container?

Comment (akrabat) [reply README.md]: I'm trying to discourage use of Environment, request or response in the container as they won't be there in 4.

Comment (ClintDavis) [reply README.md]: note - there is a colon and not a semicolon at the end of line 34 in the readme file

Comment (geggleto) [reply README.md]: oh noes!

Comment (akrabat) [reply README.md]: Doh! Fixed & push.

**Pull request merged**